chrome/content/zotero/preferences/preferences_sync.js

```diff
@@ -213,9 +213,9 @@ Zotero_Preferences.Sync = {
 	updateStorageSettingsGroups: function (enabled) {
 		var storageSettings = document.getElementById('storage-settings');
 		var menulists = storageSettings.getElementsByTagName('menulist');
-		for each(var menulist in menulists) {
-			if (menulist.className == 'storage-groups') {
-				menulist.disabled = !enabled;
+		for (let i=0; i<menulists.length; i++) {
```

Comment:
Do you prefer `for...of` here? I thought that the loop here is more consistent to some other loops in the `zotero` code: https://github.com/zotero/zotero/search?utf8=%E2%9C%93&q=getElementsByTagName